sherlock/internal/pactbroker/record_deployment.go

```diff
@@ -0,0 +1,43 @@
+package pactbroker
+
+import (
+	"errors"
+	"fmt"
+	"github.com/broadinstitute/sherlock/sherlock/internal/config"
+	"github.com/google/uuid"
+	"github.com/rs/zerolog/log"
+	"net/http"
+)
+
+const pburl = `https://pact-broker.dsp-eng-tools.broadinstitute.org`
+
+// Record deployment to pact broker
+// https://docs.pact.io/pact_broker/recording_deployments_and_releases
+func RecordDeployment(chartName string, appVersion string, eID uuid.UUID) {
+	if config.Config.Bool("pactbroker.enabled") {
+
+		request, err := http.NewRequest(http.MethodPost, pburl+"/pacticipants/"+chartName+
+			"/versions/"+appVersion+"/deployed-versions/environment/"+eID.String(), nil)
+		if err != nil {
+			PactSwallowErrors(err)
+		}
+		request.Header.Set("Content-Type", "application/json; charset=utf-8")
+		request.Header.Set("Accept", "application/hal+json")
+		request.SetBasicAuth(config.Config.MustString("pactbroker.auth.username"), config.Config.MustString("pactbroker.auth.password"))
```

Comment:
```suggestion
		request.SetBasicAuth(config.Config.String("pactbroker.auth.username"), config.Config.String("pactbroker.auth.password"))
```
I don't think this function should use `MustString`, because that will panic and crash the whole request if the config value isn't present. `String` will just result in an empty string, so we should get an authentication error -- which will probably be more helpful than a panic anyway.